### PR TITLE
Feedtag test

### PR DESCRIPTION
**Pull request by sviande**
Hi this PR:
* Add test for feedTag.Next()
* Add info into contributing.md for launching tests

### CODE REVIEW

File: feeds.go [hunk 1]
```diff
@@ -131,10 +131,10 @@ func (ft *FeedTag) Next() bool {
 		},
 	)
 	if err == nil {
-		newFT := FeedTag{}
-		err = json.Unmarshal(body, &newFT)
+		newFT := &FeedTag{}
+		err = json.Unmarshal(body, newFT)
 		if err == nil {
-			*ft = newFT
+			*ft = *newFT
```
Comment (ahmdrz) [inline feeds.go]: I have a question. What's the difference between `*ft = *newFT` and `ft = newFT`? Both of them are a pointer of FeedTag. So I think you don't need `*` for it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,3 +6,30 @@ If you want to contribute to Goinsta API you must follow a simple instructions.
 
 - **Test your code after making pull request**. The title says it all.
 - **Include jokes if you can**. This instruction is optional.
+
+# Tests
```
Comment (ahmdrz) [inline CONTRIBUTING.md]: Such a helpful thing :))
Please use `###` instead of `#`.

**Comment by ahmdrz** [comment]
Could you please fix the `travis` problem? 
#240

File: feeds.go (hunk 1 above)
Comment (sviande) [reply feeds.go]: Yes both of them are pointer, they have the same type but we need *ft to update inner value of feedTag.
Please view attached playground https://play.golang.org/p/jVG7TjccB8W

**Pull request merged**